PipeMethodCalls/Endpoints/PipeServer.cs

```diff
@@ -33,6 +33,17 @@ public PipeServer(string pipeName, Func<THandling> handlerFactoryFunc, PipeOptio
 			this.options = options;
 		}
 		
+		/// <summary>
+		/// Initializes a new instance of the <see cref="PipeServer{THandling}"/> class.
+		/// </summary>
+		/// <param name="existingPipe">Instance of a named pipe to upgrade.</param>
+		/// <param name="handlerFactoryFunc">A factory function to provide the handler implementation.</param>
+		public PipeServer(NamedPipeServerStream existingPipe, Func<THandling> handlerFactoryFunc)

```

Comment:
Instead of "existingPipe", I would name this parameter "rawPipe" to correspond with the property it's exposed under after creating.

Reply:
Will do